client/web/compose/src/components/ModuleFields/Editor/Bool.vue

```diff
@@ -21,8 +21,8 @@
           {{ label }}
         </span>
 
-        <hint
-          :text="hint"
+        <c-hint
```

Comment:
You have to remove all traces of the old hint component including imports and registrations